t/12-bmwqemu.t

```diff
@@ -17,6 +17,7 @@ use Mojo::File qw(path);
 use Mojo::JSON qw(decode_json);
 use Cpanel::JSON::XS ();
 use Test::Warnings qw(warning :report_warnings);
+use YAML::PP;
```

Comment:
Since bmwqemu does already load the module, and it isn't used directly in the test, it isn't needed here.